jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/complex/Pagination.java

```diff
@@ -0,0 +1,29 @@
+package com.epam.jdi.light.ui.bootstrap.elements.complex;
+
+import com.epam.jdi.light.common.JDIAction;
+import com.epam.jdi.light.elements.base.UIListBase;
+import com.epam.jdi.light.elements.common.UIElement;
+import com.epam.jdi.light.ui.bootstrap.asserts.PaginationAssert;
+
+public class Pagination extends UIListBase<PaginationAssert> {
+
+    @JDIAction("Check '{name}'")
+    public void click() {
+        uiElement.check();
+    }
+
+    @JDIAction("Select '{0}' in '{name}'")
+    public UIElement get(int index) {
+        return list().get(index - 1);
+    }
+
+    @JDIAction("Select '{0}' in '{name}'")
+    public void click(int index) {
+        list().select(index - 1); }
```

Comment:
} should be in the new line (you can use ctrl-alt-L / cmd-alt-L to fix it by Intellij Idea automatically)

Reply:
Fixed.